data/mods/Magiclysm/monsters/Orcs.json

```
@@ -0,0 +1,103 @@
+[
+  {
+    "id": "mon_orc_warrior",
+    "type": "MONSTER",
+    "name": { "str": "Orc Warrior" },
+    "description": "This green brute is a orc warrior. It stares at you menacingly, club in hand.",
```

```suggestion
    "description": "This green brute is a orc warrior.  It stares at you menacingly, sword in hand.",
```
Easy to forget (I do/did) - the MANUAL_OF_STYLE.md file, as checked by various things, requires 2 spaces after each period.